src/client/Microsoft.Identity.Client/ConfidentialClientApplication.cs

```diff
@@ -166,6 +169,42 @@ public sealed partial class ConfidentialClientApplication
                 longRunningProcessSessionKey);
         }
 
+        /// <summary>
+        /// Stops an in progress long running OBO session by removing the tokens associated with the provided cache key.
+        /// </summary>
+        /// <param name="longRunningProcessSessionKey">OBO cache key used to remove the tokens</param>
+        /// <param name="cancellationToken">Cancellation token</param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentNullException">Thrown if <paramref name="longRunningProcessSessionKey"/> is not set.</exception>
+        public async Task StopLongRunningWebApiAsync(string longRunningProcessSessionKey, CancellationToken cancellationToken = default)
```

Comment:
Should this return a boolean ? Simialr to List.Remove - https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.remove?view=net-7.0

Reply:
Not sure, there are 2 lists where the tokens are removed from so there would be two Boleen values. Also, Remove account does not do this and this code is based on that logic.

Reply:
If any token is removed, return true.